src/App.jsx

```diff
@@ -14,33 +16,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_ALPHABIT = 'alphabet';
+const SORT_BY_LENGTH = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getPreparedGoods(goods, { sortField, reverse }) {
+  let preparedGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  preparedGoods.sort((good1, good2) => {
+    switch (sortField) {
+      case SORT_BY_ALPHABIT:
+        return good1.localeCompare(good2);
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+      case SORT_BY_LENGTH:
+        return good1.length - good2.length;
+
+      default:
+        return 0;
+    }
+  });
+
+  if (reverse) {
+    preparedGoods = preparedGoods.toReversed();
```

Comment:
The method `toReversed()` is not a standard JavaScript method for arrays. You should use `reverse()` instead to reverse the array in place. Note that `reverse()` mutates the array, so you might want to create a copy of the array before reversing it if you need to preserve the original order.